doc/datavolume-claim-adoption.md

```diff
@@ -0,0 +1,52 @@
+# DataVolume Claim Adoption
+
+## Summary
+
+Historically, it has been an error to create a `DataVolume` with the same name as an existing `PersistentVolumeClaim`.
+
+```bash
+✗ k get dv dv1
+Error from server (NotFound): datavolumes.cdi.kubevirt.io "dv1" not found
+
+k get pvc dv1
+NAME   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      AGE
+dv1    Bound    pvc-de8b1186-fdd1-4372-8135-9c904ecf73bc   7Gi        RWX            rook-ceph-block   9m7s
+
+✗ k create -f dv.yaml
+Error from server: error when creating "dv3.yaml": admission webhook "datavolume-validate.cdi.kubevirt.io" denied the request:  Destination PVC default/dv1 already exists
```

Comment:
Where did dv3.yaml come from? the example is calling `-f dv.yaml`

Reply:
ha good catch locally my file was named dv3.yaml